components/styled/icon/src/atoms/cross-icon.tsx

```diff
@@ -1,18 +1,38 @@
 import { FC, HTMLAttributes } from 'react'
-import { calculateIconSize, IconSize } from '@ltht-react/styles'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { calculateIconSize, ICON_COLOURS, IconSize, IconStatus } from '@ltht-react/styles'
+import { FontAwesomeIcon, FontAwesomeIconProps } from '@fortawesome/react-fontawesome'
 import { faTimes } from '@fortawesome/free-solid-svg-icons'
+import { SerializedStyles, css } from '@emotion/react'
+import styled from '@emotion/styled'
 
-const CrossIcon: FC<ExternalLinkIconProps> = ({ size, className, ...rest }) => (
-  <FontAwesomeIcon
+const calculateIconColor = (status: string): SerializedStyles => css`
```

Comment:
status could be a TS type instead of an open string. 

Linguistically I find it odd to read "status: 'red'", I'd argue warning is a status but red is a colour. I also realise that remember that warning === red and success === green is extra cognitive load. 

How about we expand this method to accept either "success" or "green" if you want green, etc.?